contrib/hadoop-store-builder/src/java/voldemort/store/readonly/mr/azkaban/VoldemortBuildAndPushJob.java

```diff
@@ -703,7 +702,6 @@ public String runBuildStore(Props props, String url) throws Exception {
                                                             props.getBoolean(SAVE_KEYS, true),
                                                             props.getBoolean(REDUCER_PER_BUCKET, true),
                                                             props.getInt(BUILD_CHUNK_SIZE, 1024 * 1024 * 1024),
-                                                            props.getInt(NUM_CHUNKS, -1),
```

Comment:
Instead of never looking up the "num.chunks" key, could you test for it's presence and emit a "The num.chunks parameter is deprecated, ignoring" warning to the Azkaban log?